### ISSUE TITLE
favicon generation works, but "[object Promise]" is added to the body, and there is no <link>

### ISSUE THREAD

**Issue by insanity54** [atomrc/eleventy-favicon#2]
I would like to use this module to generate a favicon in my webpage, but I don't think the favicon shortcode promise is resolving correctly.  here's contents of my `<head>` within my layout file.

```html
  <head>
    {% favicon './website/favicon.svg' %}
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>{{ title or metadata.title }}</title>
    <meta name="description" content="{{ description or metadata.description }}">
    <link rel="stylesheet" href="{% webpackAsset 'main.css' %}">
  </head>
```

generated head:

```html
<head>
    </head>
```

It seems like everything in the <head> tag which follows     `{% favicon './website/favicon.svg' %}` gets shoved into body, not head. (truncated for brevity)

Also there is no `<link rel="icon" href="/favicon.ico">` or mention of .svg anywhere in the rendered html.

```html
<body data-controller="fp">[object Promise]
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>(...)</title>
    <meta name="description" content="(...)">
    <link rel="stylesheet" href="/assets/index.css">

   (...)

</body>
```

Edit: I think this has to do with [asynchronous-shortcodes](https://www.11ty.dev/docs/languages/nunjucks/#asynchronous-shortcodes) in eleventy.

**Comment by atomrc** [comment]
Ohh interesting indeed, I think you are right, I should probably use an async shortcode here. 
I wonder why I never ran into that issue.

**Comment by atomrc** [comment]
Fixed with version `1.1.2`. Thanks for the report
